mmv1/products/artifactregistry/Repository.yaml

```diff
@@ -736,3 +743,16 @@ properties:
     description: |-
       If true, the cleanup pipeline is prevented from deleting versions in this
       repository.
+  - name: 'vulnerabilityScanningConfig'
+    type: NestedObject
+    description: |-
+      Configuration for vulnerability scanning of artifacts stored in this repository.
+    allow_empty_object: true
```

Comment:
It looks like this field may need default_from_api because the API returns a default if its unset (which is why most of the tests are failing.) See https://googlecloudplatform.github.io/magic-modules/develop/diffs/ for more info.

Reply:
Thanks for the tip, that fixed the failures.